Source/NSActionCell.m

```diff
@@ -386,10 +387,7 @@ - (void) encodeWithCoder: (NSCoder*)aCoder
     }
   else
     {
-      int32_t tmp;
-
-      tmp = _tag;
-      [aCoder encodeValueOfObjCType: @encode(int32_t) at: &tmp];
+      encode_NSInteger(aCoder, &_tag);
```

Comment:
As you previously mentioned "long long" or any type with spaces in it will break this macro.  Is there any way to fix this?